### ISSUE TITLE
output/test/e100_valacc0.98349.pth availability?

### ISSUE THREAD

**Issue by dhillyon** [fire717/movenet.pytorch#50]
`pth2onnx.py` mentions a weights file `output/test/e100_valacc0.98349.pth` but this doesn't exist in the output folder. Could this please be made available?

**Comment by fire717** [comment]
Thank you for your interest in this repo. I apologize that I am unable to provide the model at this time as it was trained on the intranet during my previous job at the company. And that is based on our business data, with more accurate labeling and a relatively simple validation set, so the score is relatively high.

I don't know what your specific scenario is. If it's optimization for academic paper growth points, then movenet is not a good baseline because it has not published any papers or has no official open source implementation.

If it is for your own project, it is recommended that you supplement more training data. At the same time, if it is not limited to using movenet, it is now recommended that you try the newly released RTMPose.
